First 3 pages for the mobile developer guide (#10)
bcgov/mobile-developer-guide: docs/distribution_methods.md

# Mobile app distribution methods

> Mobile apps are distributed either through the public app stores or MDM InTune. Review the diagram below to determine the distribution method for your app.

Author: ```suggestion
Mobile apps are distributed through public app stores or MDM InTune. Your distribution method is determined by who will use your app.
```

@kaitlyn-rosenburg - is it ok if I added 'distributed' to the sentence? It seems like it is missing a word?

> ![Diagram of the various distribution methods. If the app is for the general public, it is distributed via the Apple App Store and Google Play Stores. If it is an app for Government Employees, it is distributed via MDM inTunes. If it is an app for Employees and Contractors it is distributed as an unlisted app via the Apple App Store](assets/distribution.drawio.svg)

Reviewer: brilliant alt text 😻

> Public apps are released through the Province's accounts on Apple's App Store and Google's Play Store. You must list your app through the Province's accounts. The developer experience team will assist you in setting up your app on these accounts.  Please refer to the [project initiation](initiate.md) page for the information needed to setup an app. Once setup you will have access to [App Store Connect](https://appstoreconnect.apple.com) and [Google Play Console](https://play.google.com/console/about/) to manage your app.

Reviewer: Probably need a link out to or description of "how to get in touch with developer experience team" here?

Author: added link

> Public apps are released through the Province's accounts on Apple's App Store and Google's Play Store. You must list your app through the Province's accounts. The [developer experience team](contact.md) will assist you in setting up your app on these accounts. Refer to the [project initiation](getting_started.md) page for the information needed to setup an app. Once setup is complete, you'll have access to [App Store Connect](https://appstoreconnect.apple.com) and [Google Play Console](https://play.google.com/console/about/) to manage your app.

Author: The name on the stores is "His Majesty the King in right of the Province of British Columbia", so I was trying to go with something a little less wordy. I know Karl is looking into changing the name, but I'm not sure the status.

Both the Apple App Store and Google Play Store have an app review process. When creating your app, it's important to consider how the app reviewer will interact with your app. For instance, if your app requires a login, the app reviewer will need either an account or access to a demo account. If your internal app can't go through the app review process, inform the [developer experience team](contact.md).

Please consult [Apple's app guidelines](https://developer.apple.com/app-store/review/) and [Google's app guidelines](https://support.google.com/googleplay/android-developer/answer/9859455?hl=en&ref_topic=7072031&sjid=10634496881788336983-NA) when preparing your app's release. It's important to follow their guidelines to ensure a smooth app review process.

> Employee apps are distributed through [MDM inTune by the OCIO Device Management Team](https://citz.sp.gov.bc.ca/sites/ES/DS/MDAS/Docs/SitePages/Home.aspx). While historically, employee apps only supported iOS devices, they're now also supported by Android. If you're planning to create an internal Android app, [contact us](contact.md). You'll be the first app and will need extra setup.

Author: ```suggestion
Employee apps are distributed through [MDM inTune by the OCIO Device Management Team](https://citz.sp.gov.bc.ca/sites/ES/DS/MDAS/Docs/SitePages/Home.aspx). While historically, employee apps only supported iOS devices, they're now also supported by Android. If you're planning to create an internal Android app, [contact us](contact.md). 
```

@kaitlyn-rosenburg - I removed the part about being the first app

> Employee and contractor apps also need extra setup in Apple's App Store Connect. The developer experience team will assist in this setup.

Reviewer: we might want a link here as well for those folks that dip in and out of parts of docs and don't read them from top to bottom.

Author: Added a link to the contact page
